### PR TITLE
#168378655 integrating LFA feedback 

### PR DESCRIPTION
**Pull request by victor-abz**
#### What does this PR do?
 >Use destructuring in server/db/index.js
> Remove unnecessary comments
> Remove Commented out codes
> No unnecessary comments
> Removing Provilege typo to Privilege

#### How should this be manually tested?
Look for changes in the diffs by git.
If want to test functionality, please clone the branch to your machine and run the following
```
npm install
```
and then test endpoints with Postman

#### What are the relevant pivotal tracker stories?
#168378655

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DATABASE_URL=postgres://victor:1234@127.0.0.1:5432/free_mentors
+
```
Comment (njeri-ngigi) [inline .env:2]: It's not recommended to commit the `.env` file to git. Instead, create another file called `.env.sample` a file that looks like the `.env` file but replace the actual data with dummy data. And commit the `env.sample` file instead.

File: server/controllers/userAuthController.js [hunk 2]
```diff
@@ -1,109 +1,18 @@
-import userModel from '../models/userAuth';
+// import userModel from '../models/userAuth';
 import Helper from '../helpers/helper';
+import Db from '../db';
 
 const userAuthController = {
+  createUser: (req, res) => new Db().addUser(req.body, res),
```
Comment (njeri-ngigi) [inline server/controllers/userAuthController.js:6]: its also not recommended to keep passing down res. The controller should be the one handling receiving a request & returning a response. Instead let Db().addUser return something from the db and then let the controller return failure (400 | 500) or success (201)

**Review by njeri-ngigi — COMMENTED** [review]
Remember to remove commented out code

**Comment by victor-abz** [comment]
This PR was Covered by #44 and #45 so there is no need to merge it.

**Pull request closed without merging**